src/main/java/helpers/ElementHelper.java

```diff
@@ -10,13 +12,25 @@ public static void scrollToView(WebElement element) {
         js.executeScript("arguments[0].scrollIntoView();", element);
     }
 
-    public static void selectDropdownByText(WebElement element, String text){
+    public static void selectDropdownByText(WebElement element, String text) {
         Select dropdown = new Select(element);
         dropdown.selectByVisibleText(text);
     }
 
-    public static void selectDropdownByIndex(WebElement element, Integer index){
+    public static void selectDropdownByIndex(WebElement element, Integer index) {
         Select dropdown = new Select(element);
         dropdown.selectByIndex(index);
     }
+
+    public static boolean doesElementExist(By element) {
+        return BrowserHelper.getDriver().findElements(element).size() != 0;
+    }
```

Comment:
@logigear-thinhho why needs this while you had isDisplayed() method below?

Reply:
I wanna still put it in ElementHelper and make it like a framework to use for another testcases after.